core/jobs/batch_jobs/audit_exploration_jobs.py

```diff
@@ -0,0 +1,1479 @@
+            for component_tag in component_tags:
+                customization_args = {}
+                for attr in tag_attrs:
+                    try:
+                        # Unescape special HTML characters such as '&quot;'.
+                        attr_val = html.unescape(component_tag[attr])
+                        customization_args[attr] = json.loads(attr_val)
+                    except Exception:
+                        continue
+
+                component = {
+                    'id': tag_name,
+                    'customization_args': customization_args
+                }
+                components.append(component)
+        return components
+
+    @staticmethod
+    def filter_invalid_rte_image_tag(
+        states_dict: Dict[str, state_domain.State]
+    ) -> List[Dict[str, List[Dict[str, List[str]]]]]:
+        """Returns the errored state values, validates
+            - alt should have at least 5 characters
+            - Image should have an SVG extension
+
+        Args:
+            states_dict: dict[str, State]. The dictionary containing
+                state name as key and State object as value.
+
+        Returns:
+            states_with_values: list[dict]. The list of dictionaries
+            containing the errored values.
+        """
+        states_with_values = []
+
+        for state_name, state in states_dict.items():
+            rte_image_errors = []
+            rte_components = ExpStateAuditChecksJob._get_rte_components(
+                state.content.html)
+            for idx, rte_component in enumerate(rte_components):
+                if rte_component['id'] == 'oppia-noninteractive-image':
+                    try:
+                        alt_with_value = rte_component['customization_args'][
+                            'alt-with-value']
+                    except Exception:
+                        alt_with_value = 'Not found'
+
+                    # Validates if alt value is greater than 5.
+                    if (
+                        alt_with_value != 'Not found' and
+                        len(alt_with_value) < 5
+                    ):
+                        rte_image_errors.append(
+                            f'State - {state_name} Image tag alt value '
+                            f'is less than 5 at index {idx} '
+                            f'having value {alt_with_value}.'
+                        )
+                    try:
+                        file_with_value = rte_component['customization_args'][
+                            'filepath-with-value']
+                    except Exception:
+                        file_with_value = 'Not found'
+
+                    # Validates filepath extension is svg.
+                    if (
+                        file_with_value != 'Not found' and
+                        len(file_with_value) > 0 and
+                        file_with_value[-3:] != 'svg'
+                    ):
```

Comment:
File name should end with `.svg`:

```suggestion
                    if (
                        file_with_value != 'Not found' and
                        len(file_with_value) > 4 and
                        file_with_value[-4:] != '.svg'
                    ):
```

Reply:
Done